bundle/tests/scorecard/kuttl/trace/01-trace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: openliberty.io/v1beta1
+kind: OpenLibertyTrace
+metadata:
+  name: example-trace
+spec:
+  podName: my-liberty-app-0
+  traceSpecification: "*=info:com.ibm.ws.webcontainer*=all"
+  maxFileSize: 20
+  maxFiles: 5
```

Comment:
@yongja79 Looks good. Thanks. Few enhancements:

- Ensure `disable: false` is set by default (new requirement from https://github.com/OpenLiberty/open-liberty-operator/pull/239)
- Set `disable: true` and make sure status reflects that tracing was disabled
- Change replicas to 2 when OpenLibertyApplication is created. After the first test, change pod name from my-liberty-app-0 to my-liberty-app-1. Status should state the trace is enabled on the latter (also indicates that tracing was stopped on the former pod)